EIPS/eip-6049.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: 6049
+title: Deprecate SELFDESTRUCT
+description: Deprecate SELFDESTRUCT by discouraging its use and warning about a potential future behavior change.
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/deprecate-selfdestruct/11907
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-11-27
+---
+
+## Abstract
+
+This EIP deprecates the `SELFDESTRUCT` opcode and warns against its use. A breaking change to this functionality is likely to come in the future.
```

Comment:
Would it be useful to link out to https://ethereum-magicians.org/t/eip-4758-deactivate-selfdestruct/8710 here for additional context? Or is it bad practice to have one EIP link to another?

Reply:
I understand that linking out is sometimes frowned upon. Please discussion in the discussion thread if you would like to raise this. It is above my pay grade.

Reply:
SGTM–no strong opinion here, was curious and wanted to flag! 👍 💜